ONSdigital/ras-rm-secure-message-v2
# Info endpoint

# What and why?
This PR creates the /info endpoint for this service using flask.

# How to test?
Clone this repository, do a pipenv install, and then do `python run.py`, which should run the service on port 5051. Navigate to `localhost:5051/info` and you should hit the endpoint okay and be able to see the info.

# Jira
[Card](https://jira.ons.gov.uk/browse/RAS-297)
